### PR TITLE
Add language fr (French translation)

### PR DESCRIPTION
**Pull request by matthieukern**
Just wrote a French translation to this awesome beginner tutorial for use at the French's Senate later this month as support for a workshop with some senators, so I share it with you :)

### CODE REVIEW

**Comment by sethvincent** [comment]
@matthieukern That's awesome! Is there someone that could do a review of the translation before we merge it in?

File: problems/accessing-array-values/problem_fr.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+```js
+var pets = ['cat', 'dog', 'rat'];
+
+console.log(pets[0]);
+```
+
+Le code ci-dessus affichera le premier élément du tableau `pets` - la chaine de caracteres `cat`.
+
+Les éléments de tableaux doivent uniquement être accedées au travers de la notation des crochets.
```
Comment (CapucineTroalen) [inline problems/accessing-array-values/problem_fr.md]: On ne doit accéder aux éléments de tableaux au travers de la notation des crochets.

**Comment by matthieukern** [comment]
@sethvincent: @vitar67 just did a great job reviewing and correcting the entire translation ; I think it is ok now!

**Comment by sethvincent** [comment]
Cool, thank you for this!

**Pull request merged**